siliconcompiler/toolscripts/rhel9/install-xyce.sh

```
@@ -19,15 +22,20 @@ if [ -z ${PREFIX} ]; then
-# Install core dependencies.
-sudo yum install -y gcc gcc-c++ gcc-gfortran blas blas-devel \
+# Core dependencies. The 'devel' repository is disabled by default; enabling and
+# disabling it is root-only work whose only purpose is the install between them,
+# so the whole block is skipped when those packages are already present.
+devel_pkgs="gcc gcc-c++ gcc-gfortran blas blas-devel \
     cmake lapack lapack-devel bison flex fftw-devel fftw \
     suitesparse suitesparse-devel autoconf automake libtool \
-    git
-sudo dnf config-manager --set-disabled devel || true
+    git"
+if prereqs_missing $devel_pkgs; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)